modules/checklist/assets/js/app/components/checklist-wrapper.js

```diff
@@ -2,27 +2,33 @@ import { useState } from 'react';
 import { List } from '@elementor/ui';
 import CheckListItem from './checklist-item';
 import PropTypes from 'prop-types';
+import SuccessMessage from './success-message';
+import { isStepChecked } from '../../utils/functions';
 
 const ChecklistWrapper = ( { steps, setSteps } ) => {
 	const [ expandedIndex, setExpandedIndex ] = useState( -1 );
+	const checkIfAllStepsCompleted = steps.filter( isStepChecked ).length === steps.length;
 
 	return (
-		<List component="div" sx={ { py: 0 } }>
-			{
-				steps.map( ( step, index ) => {
-					return (
-						<CheckListItem
-							key={ index }
-							step={ step }
-							setSteps={ setSteps }
-							setExpandedIndex={ setExpandedIndex }
-							expandedIndex={ expandedIndex }
-							index={ index }
-						/>
-					);
-				} )
-			}
-		</List>
+		<>
+			<List component="div" sx={ { py: 0 } }>
+				{
+					steps.map( ( step, index ) => {
+						return (
+							<CheckListItem
+								key={ index }
+								step={ step }
+								setSteps={ setSteps }
+								setExpandedIndex={ setExpandedIndex }
+								expandedIndex={ expandedIndex }
+								index={ index }
+							/>
+						);
+					} )
+				}
+			</List>
+			{ checkIfAllStepsCompleted ? <SuccessMessage /> : null }
+		</>
```

Comment:
I don't think that both the step list and the success message should be displayed at the same time
```suggestion
		checkIfAllStepsCompleted
			? <SuccessMessage />
			:  <List component="div" sx={ { py: 0 } }>
				{
					steps.map( ( step, index ) => {
						return (
							<CheckListItem
								key={ index }
								step={ step }
								setSteps={ setSteps }
								setExpandedIndex={ setExpandedIndex }
								expandedIndex={ expandedIndex }
								index={ index }
							/>
						);
					} )
				}
			</List>
```